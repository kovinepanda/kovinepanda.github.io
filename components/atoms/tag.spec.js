import { mount } from "@vue/test-utils";
import Tag from './tag';

describe('Tag', () => {
    it('renders correctly', () => {
        const wrapper = mount(Tag, {
            propsData: {
                slug: 'vue ðŸ¤˜'
            }
        })
        expect(wrapper.element).toMatchSnapshot()
      })
})
