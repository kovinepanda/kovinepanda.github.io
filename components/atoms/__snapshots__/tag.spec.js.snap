// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag renders correctly 1`] = `
<nuxt-link
  class="contrast outline"
  to="/tags/vue ðŸ¤˜"
>
  
  #vue ðŸ¤˜

</nuxt-link>
`;
